package.json

```diff
@@ -9,11 +9,11 @@
     "wpt": "grunt wpt",
     "fix": "grunt fix",
     "unittest": "grunt unittest",
-    "gen_wpt_cts_html": "tools/gen_wpt_cts_html",
-    "dev": "tools/dev_server"
+    "gen_wpt_cts_html": "node tools/gen_wpt_cts_html",
```

Comment:
`node` is required to use these in windows if you don't have ".js" associated with node.

Reply:
oops, thanks for catching this! Are things otherwise working well on Windows? I haven't tried in, like, years.